libnavigation-core/build.gradle

```diff
@@ -5,6 +5,7 @@ apply plugin: 'org.jetbrains.dokka'
 apply plugin: 'com.jaredsburrows.license'
 apply plugin: 'com.mapbox.android.sdk.versions'
 apply from: "${rootDir}/gradle/ktlint.gradle"
+apply from: file("${rootDir}/gradle/artifact-settings.gradle")
```

Comment:
nit: are there other options for rootDir? finding that it makes it difficult to include in other repositories (nit because we're not really supporting that here)

Reply:
I see what you mean, especially when you include this as a submodule - I suffer that in the past. We can address in a separate PR if needed 👍